Networking/Networking/Model/ShippingLineTax.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+/// Represents the taxes for a specific shipping item.
+///
+public struct ShippingLineTax: Decodable, Hashable {
+
+    /// Tax ID for shipping item
+    ///
+    public let taxID: Int64
+
+    /// Tax subtotal
+    ///
+    public let subtotal: String
+
+    /// Shipping tax amount
+    ///
+    public let total: String
```

Comment:
it's interesting that I cannot find these fields in the [WC API doc](https://woocommerce.github.io/woocommerce-rest-api-docs/#order-taxes-properties) 🤔

Reply:
Yeah, documentation is not great :-(